llvm/test/CodeGen/SPIRV/hlsl-intrinsics/all.ll

```diff
@@ -1,5 +1,5 @@
-; RUN: llc -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s --check-prefixes=CHECK,CHECK-HLSL
-; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s --check-prefixes=CHECK,CHECK-OCL
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s --check-prefixes=CHECK,CHECK-HLSL
```

Comment:
There are a bunch of other hlsl intrinsic tests, what was the requirement for `-verify-machineinstrs` on these 3 tests but not on the other tests in this folder?

Reply:
The ultimate goal is to have `-verify-machineinstrs` set for all tests in our test suite where it's possible. At the moment not all tests are passing with `-verify-machineinstrs`, so not all hlsl intrinsic tests can be updated right now. Also I didn't want to include all ~180 passing with `-verify-machineinstrs` LIT tests into this initial PR.